src/crc32c.cpp

```diff
@@ -46,7 +46,7 @@ namespace libtorrent
 {
 	boost::uint32_t crc32c_32(boost::uint32_t v)
 	{
-#if TORRENT_HAS_SSE
+#if (TORRENT_HAS_SSE && defined _MSC_VER && _MSC_VER >= 1600)
```

Comment:
it seems cleaner to just consider _MSC_VER < 1600 to not have SSE, don't you think? that would be a single place to change